package.json

```
@@ -24,12 +50,14 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "emnapi": "^0.43.1",
     "lodash": "^4.17.15",
     "mocha": "^5.2.0"
   },
   "dependencies": {
+    "@emnapi/runtime": "^0.43.1",
     "@mapbox/node-pre-gyp": "^1.0.8",
-    "node-addon-api": "^1.6.3",
+    "node-addon-api": "^7.0.0",
```

Maybe this could be upgraded to `node-addon-api@^8.0.0`

```suggestion
    "node-addon-api": "^8.0.0",
```

The [release notes](https://github.com/nodejs/node-addon-api/releases/tag/v8.0.0) say that the only Notable Change is:

- Support for Node.js v16.x is no longer maintained.